framework/meta.yaml

```diff
@@ -11,7 +11,8 @@ source:
 
 requirements:
   build:
-    - boost {{ boost }}
+    - _openmp_mutex {{ _openmp_mutex }}
```

Comment:
```suggestion
    - _openmp_mutex {{ _openmp_mutex }} [linux64]
```
This is only a linux runtime dependency I believe

Reply:
Good call, thanks! Done!